2 dz/6/numberGenerator.cpp

```diff
@@ -0,0 +1,32 @@
+#include "numberGenerator.h"
+
+void numberGenerator(int *hiddenNumber, int size)
+{
+    int i = 0;
+    while (true)
+    {
+        srand(time(NULL));
```

Comment:
As I said, you need to seed your generator only once, at the very beginning. It will work properly.